### PR TITLE
Update numpy@1.16

### PR DESCRIPTION
**Pull request by kkdd**
Update to numpy-1.16.6.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/numpy@1.16.rb [hunk 1]
```diff
@@ -1,16 +1,8 @@
 class NumpyAT116 < Formula
   desc "Package for scientific computing with Python"
   homepage "https://www.numpy.org/"
-  url "https://files.pythonhosted.org/packages/d3/4b/f9f4b96c0b1ba43d28a5bdc4b64f0b9d3fbcf31313a51bc766942866a7c7/numpy-1.16.4.zip"
-  sha256 "7242be12a58fec245ee9734e625964b97cf7e3f2f7d016603f9e56660ce479c7"
-  revision 1
-
-  bottle do
```
Comment (SMillerDev) [inline Formula/numpy@1.16.rb]: Please don't remove the bottle block

File: Formula/numpy@1.16.rb [hunk 2]
```diff
@@ -1,16 +1,8 @@
...
-    cellar :any
-    sha256 "6c80a9d89e01c944f4142c95f7239d796c9e0a4915c395e99a30d8b221c89280" => :catalina
-    sha256 "eae10fb059b9246e2e106572c0534b33fe5848380b945fb2afe90600b0dc5cb1" => :mojave
-    sha256 "d7713fb06785ea6dc8ff901858b7c5155f4ca5624d4c5d380b58c1f254ac6268" => :high_sierra
-  end
+  url "https://github.com/numpy/numpy/releases/download/v1.16.6/numpy-1.16.6.zip"
+  sha256 "e5cf3fdf13401885e8eea8170624ec96225e2174eb0c611c6f26dd33b489e3ff"
```
Comment (zbeekman) [inline Formula/numpy@1.16.rb]:
```suggestion
  sha256 "e5cf3fdf13401885e8eea8170624ec96225e2174eb0c611c6f26dd33b489e3ff"
  
  bottle do
    cellar :any
    sha256 "6c80a9d89e01c944f4142c95f7239d796c9e0a4915c395e99a30d8b221c89280" => :catalina
    sha256 "eae10fb059b9246e2e106572c0534b33fe5848380b945fb2afe90600b0dc5cb1" => :mojave
    sha256 "d7713fb06785ea6dc8ff901858b7c5155f4ca5624d4c5d380b58c1f254ac6268" => :high_sierra
  end  
```

File: Formula/numpy@1.16.rb [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
+  
   bottle do
     cellar :any
     sha256 "6c80a9d89e01c944f4142c95f7239d796c9e0a4915c395e99a30d8b221c89280" => :catalina
     sha256 "eae10fb059b9246e2e106572c0534b33fe5848380b945fb2afe90600b0dc5cb1" => :mojave
     sha256 "d7713fb06785ea6dc8ff901858b7c5155f4ca5624d4c5d380b58c1f254ac6268" => :high_sierra
-  end
+  end
```
Comment (zbeekman) [inline Formula/numpy@1.16.rb]:
```suggestion
  end
```

**Comment by zbeekman** [comment]
@SMillerDev @dawidd6 @samford I'm at a loss as to whether or not the (now system) python 2 resources need updating and how to accomplish that. I cannot for the life of me get homebrew-pypi-poet and numpy to play nice with system python

**Comment by kkdd** [comment]
```pip2``` says that ```numpy 1.16.6``` is the latest as of now. I don't know, however, whether it is the last version supporting python 2 or not.

```console
$ pip2 list --outdated
Package   Version    
-------   -------
numpy     1.16.4  1.16.6 wheel
```

**Comment by Bo98** [comment]
Yes 1.16.x is the last for Python 2.

Here's an updated Cython resource:

```rb
  resource "Cython" do
    url "https://files.pythonhosted.org/packages/d9/82/d01e767abb9c4a5c07a6a1e6f4d5a8dfce7369318d31f48a52374094372e/Cython-0.29.15.tar.gz"
    sha256 "60d859e1efa5cc80436d58aecd3718ff2e74b987db0518376046adedba97ac30"
  end
```

**Review by zbeekman — APPROVED** [review]
LGTM

Thanks for your contribution to Homebrew! 🎉 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you @kkdd!

**Pull request closed without merging**